### PR TITLE
Bump aiokatcp version

### PR DESCRIPTION
**Pull request by amishatishpatel**
As per the release of aiokatcp version 2.0.0. 

Checklist (if not applicable, edit to add `(N/A)` and mark as done):

- [x] If dependencies are added/removed: update `setup.cfg` and `.pre-commit-config.yaml`
- [x] (n/a) If modules are added/removed: use `sphinx-apidoc -efo doc/ src/` to update files in `doc/`
- [x] Ensure copyright notices are present and up-to-date
- [x] (n/a) If qualification tests are changed: attach a sample qualification report
- [x] (n/a) If design has changed: ensure documentation is up to date
- [x] (n/a) If ICD-defined sensors have been added: update `fake_servers.py` in katsdpcontroller to match

Closes: NGC-1062.

### CODE REVIEW

**Comment by amishatishpatel** [comment]
@bmerry I'm going to resolve the corresponding katsdpcontroller update and send them both to you for review at the same time.

File: setup.cfg [hunk 1]
```diff
@@ -15,7 +15,7 @@ package_dir =
     = src
 packages = find:
 install_requires =
-    aiokatcp>=1.9.0
+    aiokatcp>=2.0.0
```
Comment (bmerry) [inline setup.cfg:18]: I'd revert this one. The package should still work fine with 1.9.0, and if someone wants to install it that way we shouldn't stop them; we just choose to deploy with 2.0.0 ourselves.

**Review by bmerry — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (amishatishpatel) [reply setup.cfg:18]: Ah, noted!

**Review by bmerry — APPROVED** [review]
LGTM. It's independent of the katsdpcontroller update, so feel free to merge.

**Pull request merged**